OpenRA.Mods.Common/Traits/SpawnActorOnDeath.cs

```diff
@@ -50,16 +50,17 @@ public class SpawnActorOnDeathInfo : ITraitInfo
 			"lead to unexpected behaviour.")]
 		public readonly CVec Offset = CVec.Zero;
 
-		public object Create(ActorInitializer init) { return new SpawnActorOnDeath(init, this); }
+		public override object Create(ActorInitializer init) { return new SpawnActorOnDeath(init, this); }
 	}
 
-	public class SpawnActorOnDeath : INotifyKilled
+	public class SpawnActorOnDeath : ConditionalTrait<ConditionalTraitInfo>, INotifyKilled
```

Comment:
It was correct before: ConditionalTrait<SpawnActorOnDeathInfo>.
Remove `readonly SpawnActorOnDeathInfo info;` and use only Info in SpawnActorOnDeath , which is defined in ConditionalTrait.